### ISSUE TITLE
!include is not supported

### ISSUE THREAD

**Issue by nicokant** [KyleKing/copier-template-tester#31]
**Describe the bug**
ctt - version 2.1.1

Using include in my copier.yaml is not supported, but copier supports it.

```
yaml.composer.ComposerError: expected a single document in the stream
```

**To Reproduce**
use the directive `!include` in the copier.yaml

**Expected behavior**
Error not raised

**Comment by KyleKing** [comment]
Thanks for reporting this issue and this should be an easy fix.

I'm planning on using the example from <https://copier.readthedocs.io/en/stable/configuring/#include-other-yaml-files> for unit testing, but let me know if there are other edge cases that would be worthwhile to test

```yaml
---
# Copier will load all these files
!include shared-conf/common.*.yml

# These 3 lines split the several YAML documents
---
# These two documents include common questions for these kind of projects
!include common-questions/web-app.yml
---
!include common-questions/python-project.yml
---

# Here you can specify any settings or questions specific for your template
_skip_if_exists:
    - .password.txt
custom_question: default answer
```

**Comment by KyleKing** [comment]
Should be fixed now with v2.1.2!
